zenodo/modules/records/serializers/derived.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```

Comment:
"Derived" metadata sounds too specific to our use-case, i.e. not all metadata are "derived" from their subject. 

I think the namespace (and other places in the code) should be renamed to either `supplementary_metadata` or `metafiles` (or something else?). I like `metafiles`, since it's short and sounds like a "branded" feature. @lnielsen WDYT?